src/requests/team.ts

```diff
@@ -111,7 +109,6 @@ export async function createTeam(
             name: $name
             slug: $slug
             domain: $domain
-            submission_email: $submissionEmail
```

Comment:
nit: I think this will break e2e tests when merged to planx-new - do you want to wait to _remove_ these fields until the current DB column is also ready to be dropped? 

I think only _additive_ changes in this first type update might be easier to manage and better match the additive-only change you've pushed to planx-new data models?

Reply:
Good shout! I'll revert this change and keep it additive-only.